Junction/Junction Example/AppDelegate.swift

```diff
@@ -23,13 +23,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             .addRow(stringSetting)
             .addRow(intSetting)
         
-        let secondSection = StringSingleSelectionSection(possibleValues: ["google.com", "yahoo.com"], enableCustom: true, name: "Single Selection String", key: "singleSelectionString")
+        let secondSection = StringSingleSelectionSection(possibleValues: ["google.com", "yahoo.com"], enableCustom: true, name: "Single Selection String", key: "singleSelectionString", defaultValue: 0)
         
-        window = JunctionWindow(frame: UIScreen.mainScreen().bounds, style: .Shake, sections: [firstSection, secondSection], enabled: true)
+        Junction.enabled = true
+        Junction.sections = [firstSection, secondSection]
+        Junction.style = .Shake
+        Junction.completionBlock = { previousValues, newValues in
```

Comment:
`completionBlock` isn't very descriptive of what it does/when it's called. `SettingsUpdatedBlock`?